docs/http.md

```diff
@@ -437,30 +437,42 @@ step.
   | geojson    | [GeoJSON `LineString`](http://geojson.org/geojson-spec.html#linestring) or [GeoJSON `Point`](http://geojson.org/geojson-spec.html#point) if it is only one coordinate (not wrapped by a GeoJSON feature)|
   
 - `name`: The name of the way along which travel proceeds.
+- `lanes`: the available turn lanes at the turn
+  - `marked`: markings on the road, following the OSM scheme (e.g. left, slight_right, or through)
```

Comment:
Well. Its really just the marking on the road. So `lane` does not really fit in my view. Would `marking` be more to your liking?